pkg/virt-controller/watch/vmi/storage.go

```diff
@@ -375,3 +375,38 @@ func (c *Controller) getFilesystemOverhead(pvc *k8sv1.PersistentVolumeClaim) (vi
 	}
 	return storagetypes.GetFilesystemOverhead(pvc.Spec.VolumeMode, pvc.Spec.StorageClassName, cdiConfig)
 }
+
+func (c *Controller) syncVolumesUpdate(vmi *virtv1.VirtualMachineInstance) {
```

Comment:
tiny nit: this belongs to the previous commit.

Reply:
done